titan/params/demographics.yml

```diff
@@ -297,12 +302,36 @@ demographics:
                 type: float
                 min: 0.0
                 max: 1.0
-              prob:
-                default: 0.0
-                description: "Probability that an agent in this class with HIV becomes enrolled in HAART at a given time step"
+              cap:
                 type: float
+                default: 0.0
+                description: "Percent of agents on haart if haart cap is used"
                 min: 0.0
                 max: 1.0
+              prob:
+                type: bin
+                description: "Probability that an agent in this class with HIV becomes enrolled in HAART at a given time step based on time since diagnosis (use 0 for haart cap)"
```

Comment:
this feels like a timeline, should it be a definition instead of bins?  I also don't love prob as the outer key since that's not how we typically use `prob`.  I'd make that the inner key instead of `enroll_prob` and figure out a better name for this that also makes it perhaps more clearly the alternate to cap

Reply:
It feels more like a bin than a def to me?

Reply:
def please!